config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
+  <<: *default
+  database: storage/development.sqlite3
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: storage/test.sqlite3
+
+production:
+  <<: *default
+  database: storage/production.sqlite3
```

Comment:
- [ ] As per the project requirements, it is essential to have a Postgres database set up instead of using sqlite3, as I noticed in your code. To address this, please update the `adapter` code at line `8` to use `postgresql` as the database adapter. Additionally, ensure that the `encoding` is set to `unicode`. Once the adapter is updated, it's crucial to execute the migrations you've created to create the required tables in the Postgres database. Assuming you have Postgres installed on your machine, running the corresponding migrations will complete the migration process and create a list of tables in the db.